src/lycophron/config.py

```diff
@@ -6,21 +6,22 @@
 # under the terms of the MIT License; see LICENSE file for more details.
 """Lycophron config classes."""
 
-import logging
 import logging
 import os
 import types
 from abc import ABC, abstractmethod
 from urllib.parse import urlparse
-from .errors import ConfigNotFound, ErrorHandler, InvalidConfig, ConfigError
+
+from .errors import ConfigNotFound, ErrorHandler, InvalidConfig
 
 logger = logging.getLogger("lycophron")
 
 
 class Defaults:
     SQLALCHEMY_DATABASE_URI = "sqlite:///lycophron.db"
-    ZENODO_URL = "https://sandbox.zenodo.org/api"
+    ZENODO_URL = "https://127.0.0.1:5000/api"
     TOKEN = "CHANGEME"
+    RECORD_BATCH_SIZE = 10
```

Comment:
Minor: I don't think this variable is being used, if not we can remove it?

Reply:
Commented it for not, it's not used but will probably be configurable in the future.